apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -173,12 +180,21 @@ class PageTemplateModal extends Component {
 				isDismissable={ false }
 				isDismissible={ false }
 			>
-				<IconButton
-					className="page-template-modal__close-button components-icon-button"
-					onClick={ this.closeModal }
-					icon="arrow-left-alt2"
-					label={ __( 'Go back' ) }
-				/>
+				{ isPromptedFromSidebar ? (
```

Comment:
Funnily enough, it seems that these buttons do not show up in the "windowed" wp-admin mode. Only in full screen mode :)

Reply:
Oh my, good catch!  That is definitely something we will want to fix...

Reply:
Fixed.  It was et up do only render on fullscreen mode in the SPT SCSS file, it looks like this was set up this way to mimic the behavior of the normal page back button at the time (which we have since overridden and forced to always show)